src/include/btmem.h

```diff
@@ -634,6 +634,8 @@ struct __wt_split_page_hist {
 struct __wt_page_block_meta {
     uint64_t page_id;
 
+    bool is_delta;
```

Comment:
The intention behind this metadata struct is to be the page's metadata, so `is_delta` is not a good name. Maybe something like `has_delta` would be more appropriate. But make we should just go all the way out and introduce `num_deltas` at this point (for the number of deltas since the last full image), as we'll need it soon anyways.